Makefile

```diff
@@ -1,6 +1,8 @@
 # SPDX-FileCopyrightText: The RamenDR authors
 # SPDX-License-Identifier: Apache-2.0
 
+include hack/make/personalvars.mk
```

Comment:
Assuming that this works when the file does not exists, we need to:

1. Document this file in the reamde or contributing document, or in new docs/developement.md.
2. Show example with the common stuff developer is likely to define